rust-tropes.md

```diff
@@ -32,6 +32,10 @@ It is a mnemonic for the HRTB acronym.
 
 The brackets, empty tuples and comma resemble an owl's head.
 
+## Pre-Poop Your Pants (PPYP)
```

Comment:
I hope that future formats will make these entries searchable by both acronym and extended form. But for the time being, acronyms are specified in the entry's body (see for example RESF).